### PR TITLE
add uninstall packages docs

### PR DESCRIPTION
**Pull request by karan**
#20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,22 @@ those that have used NPM or Bower, it is similar to your `node_modules` or
 
 ### Uninstalling Packages
 
+Atom uses a clone of `npm` called `apm` as it's package manager. (See [list of `apm` command](https://github.com/jbranchaud/splitting-atoms/blob/master/Apm.md).)
+
+You can install non-bundled packages by going to the Packages section on left hand side of the Settings view (`cmd-,`) or by using the comman `apm list`.
```
Comment (jbranchaud) [inline README.md]: Installing packages is addressed in a section above. I don't think this line is needed.

File: README.md [hunk 2]
```diff
@@ -106,6 +106,22 @@ those that have used NPM or Bower, it is similar to your `node_modules` or
...
+
+#### Uninstall a package using `apm`
+
+To uninstall a package using `apm`, simply run the command `apm uninstall <package name>`. This will remove tha package from your `~/.atom/packages/` directory.
+
+Here's a sample run of the command:
+
+    karan:splitting-atoms$ apm uninstall toggle-quotes
```
Comment (jbranchaud) [inline README.md]: For the sake of clarity, can we use the most generic command-line prompt possible? Just a `$` should work.

**Comment by karan** [comment]
My bad, fixed all.

**Comment by jbranchaud** [comment]
looks good :+1:

**Pull request merged**